### ISSUE TITLE
System tray icon invisible in flashback session

### ISSUE THREAD

**Issue by marcos-mestre** [sialan-labs/sigram#31]
I've just installed the Sigram client on my Ubuntu Trusty (14.04) with gnome-flashback-session. At first I didn't see the tray icon, but then I discovered it. The icon was there, but as a little white point. Can you fix it?
Thank you!
![image](http://i.imgur.com/5N8AQqf.jpg)

**Comment by angelbladex** [comment]
This is my icon on lxde

![icon](https://cloud.githubusercontent.com/assets/847922/3312735/b2426e02-f6d7-11e3-9889-560cb198b789.png)

**Comment by marcos-mestre** [comment]
Yes, I know that it also works fine on Unity. I don't understand the problem with Gnome Flashback.

**Comment by silberzwiebel** [comment]
The icon is missing most of the time under GNOME shell 3.12, fedora 20, too. But since other icons are missing, too, I consider this more an GNOME bug....

**Comment by realbardia** [comment]
Its because sigram use appindicator lib for systray icon on unity.
Please set DESKTOP_SESSION env to "ubuntu" and give me back result.
export DESKTOP_SESSION=ubuntu
If works fine ,I'll using appindictor on gnome too.

**Comment by marcos-mestre** [comment]
I tried closing the app, setting the variable DESKTOP_SESSION to ubuntu, running the app again, but the icon doesn't apears yet.
BTW, I don't know if it's important, but I do have an appindicat applet in the panel.

**Comment by TaylanTatli** [comment]
There is no tray icon on mate desktop too.

**Comment by marcos-mestre** [comment]
I've suffered a desktop swapping disease.
After uninstalling all the desktops, the telegram icon now appears on the systray of the flashback session. Seems that the problem was originated 'cause a missing package. Aparently.

**Comment by marcos-mestre** [comment]
After rebooting the icon dissapears again. I don't understand why this is happening. Can I send you some additional info? What do you need?

**Comment by silberzwiebel** [comment]
It's randomly (dis)appearing here, too. Maybe this message helps to debug?
`QXcbWindow: Unhandled client message: "_GTK_LOAD_ICONTHEMES"`

**Comment by realbardia** [comment]
Ok...
First install libappindicator:
sudo apt-get install libappindicator1
then set:
export DESKTOP_SESSION=ubuntu
and last, run sigram
I installed gnome3 fallback mode. this method worked for me.

**Comment by marcos-mestre** [comment]
Seems to work.
Can you changed it for not being necessary the custom value for that
variable?
Thank you.

Marcos Mestre

2014-06-28 23:40 GMT-03:00 Bardia notifications@github.com:

**Comment by TaylanTatli** [comment]
what about for Arch Mate?
